### PR TITLE
Adds a waifu-pillow box.

### PR DESCRIPTION
**Pull request by GrayRachnid**
i plan to make more waifu pillow stuff if somebody wants to sprite for me.
only downside is that you can't disassemble this
i'm not going to be sad if someone makes over 100 waifu pillows that can't be removed other than throwing them into the recycler

:cl:
rscadd: Adds a new waifu-pillow style box, available on the cardboard recipe list.
/:cl:

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
the resurrection

**Comment by Super3222** [comment]
Not again. :-1:
Incoming goof in... 3... 2... 1...

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 1]
```diff
@@ -878,4 +878,37 @@
...
+	icon_state = "pillowbox"
+	item_state = "pillowbox"
+	burn_state = -1 //nobody can burn my love
+	attack_verb = list("squished", "fluffed", "slapped", "made a smug gesture toward", "bopped", "smacked", )
+	hitsound = "sound/weapons/thudswoosh.ogg"
+	foldable = null
 
+/obj/item/weapon/storage/box/pillow/attack_self(mob/user)
+	..()
+	var/msg = pick("Uguu~", "Kyaa!", "B-baka!", "Cute!", "Aww!", "Smug!", "Kawaii!", "Desu~", "N-not there!", "Nyaa~", "Fufu~", "Sugoi!")
```
Comment (Super3222) [inline code/game/objects/items/weapons/storage/boxes.dm:893]: Did you c/p shadows old PR or are you one in the same person

Comment (GrayRachnid) [reply code/game/objects/items/weapons/storage/boxes.dm:893]: b-baka

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 2]
```diff
@@ -878,4 +878,37 @@
...
+	user.changeNext_move(CLICK_CD_MELEE)
+	playsound(loc, "sound/weapons/thudswoosh.ogg", 50, 1, -5)
+	user.visible_message("<span class='notice'>[user] squeezes \the [src].</span>","<span class='notice'>You squeeze \the [src]. [msg]</span>")
+	return
+
+/obj/item/weapon/storage/box/pillow/suicide_act(mob/user)
+	var/mob/living/carbon/human/H = user
+	user.visible_message("<span class='suicide'>[user] is cuddling \ with the [src.name]! It looks like \he's trying to commit social suicide.</span>")
+	src.say("You make my kokoro go doki-doki, [user]!")
```
Comment (Super3222) [inline code/game/objects/items/weapons/storage/boxes.dm:902]: :(

Comment (GrayRachnid) [reply code/game/objects/items/weapons/storage/boxes.dm:902]: y u sad

**Comment by Kmc2000** [comment]
Is this using my shitty pillow sprite gray?

File: code/game/objects/items/weapons/storage/boxes.dm (hunk 1 above)
Comment (ShadowDeath6) [reply code/game/objects/items/weapons/storage/boxes.dm:893]: ',:^))

**Comment by GrayRachnid** [comment]
did you make it

**Comment by GrayRachnid** [comment]
i'm making pillows not a clone of boxes, also kmc will make me a better sprite with inhands. closing this for now. I'LL BE BACK

**Pull request closed without merging**